src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import styles from './Button.module.scss'
+
+interface Props {
```

Comment:
why not to use types? https://medium.com/@koss_lebedev/type-aliases-vs-interfaces-in-typescript-based-react-apps-e77c9a1d5fd0